arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -771,55 +818,56 @@ def checkFootnote(elt, text):
             
         # 3.1.1 test
         hasOutdatedUrl = False
-        for e in outdatedTaxonomyURLs:
+        for e in val.authParam["outdatedTaxonomyURLs"]:
             if e in val.extensionImportedUrls:
                 val.modelXbrl.error("ESEF.3.1.2.incorrectEsefTaxonomyVersionUsed",
                      _("The issuer's extension taxonomies MUST import the applicable version of the taxonomy files prepared by ESMA. Outdated entry point: %(url)s"),
                     modelObject=modelDocument, url=e)
                 hasOutdatedUrl = True
                 
-        if not hasOutdatedUrl and not any(e in val.extensionImportedUrls for e in esefTaxonomyURLs):
+        if not hasOutdatedUrl and not any(e in val.extensionImportedUrls for e in val.authParam["effectiveTaxonomyURLs"]):
             val.modelXbrl.error("ESEF.3.1.2.requiredEntryPointNotImported",
                  _("The issuer's extension taxonomies MUST import the entry point of the taxonomy files prepared by ESMA."),
```

Comment:
It seems like this message needs to be updated depending on the authority.

Reply:
This still needs to be addressed

Reply:
added messages for UKFRC21.2 and UKFRC21.3